js/bybit.js

```diff
@@ -3825,170 +4385,115 @@ module.exports = class bybit extends Exchange {
+        //                     "leavesValue": "0",
+        //                     "cumExecQty": "65",
+        //                     "cumExecValue": "21.3265",
+        //                     "cumExecFee": "0.0127959",
+        //                     "lastPriceOnCreated": "0.0000",
+        //                     "rejectReason": "EC_NoError",
+        //                     "orderLinkId": "",
+        //                     "createdTime": "1657526321499",
+        //                     "updatedTime": "1657526321504",
+        //                     "orderId": "ac0a8134-acb3-4ee1-a2d4-41891c9c46d7",
+        //                     "stopOrderType": "UNKNOWN",
+        //                     "takeProfit": "0.0000",
+        //                     "stopLoss": "0.0000",
+        //                     "tpTriggerBy": "UNKNOWN",
+        //                     "slTriggerBy": "UNKNOWN",
+        //                     "triggerPrice": "0.0000",
+        //                     "closeOnTrigger": true,
+        //                     "triggerDirection": 0,
+        //                     "positionIdx": 2
+        //             ],
+        //             "nextPageCursor": "K0crQkZRL0MyQVpiN0tVSDFTS0RlMk9DemNCWHZaRHp3aFZ4Y1Yza2MyWT0="
         //         },
-        //         "time_now":"1652107028.148177",
-        //         "rate_limit_status":"598",
-        //         "rate_limit_reset_ms":"1652107028146",
-        //         "rate_limit":"600"
+        //         "retExtInfo": {},
+        //         "time": 1658899014975
         //     }
         //
         const result = this.safeValue (response, 'result', {});
-        const data = this.safeValue (result, 'data', []);
+        const data = this.safeValue2 (result, 'data', 'list', []);
         return this.parseOrders (data, market, since, limit);
     }
 
-    async fetchClosedOrders (symbol = undefined, since = undefined, limit = undefined, params = {}) {
+    async fetchOrders (symbol = undefined, since = undefined, limit = undefined, params = {}) {
         /**
          * @method
-         * @name bybit#fetchClosedOrders
-         * @description fetches information on multiple closed orders made by the user
-         * @param {string|undefined} symbol unified market symbol of the market orders were made in
+         * @name bybit#fetchOrders
+         * @description fetches information on multiple orders made by the user
+         * @param {string} symbol unified market symbol of the market orders were made in
          * @param {int|undefined} since the earliest time in ms to fetch orders for
          * @param {int|undefined} limit the maximum number of  orde structures to retrieve
          * @param {object} params extra parameters specific to the bybit api endpoint
          * @returns {[object]} a list of [order structures]{@link https://docs.ccxt.com/en/latest/manual.html#order-structure}
          */
         await this.loadMarkets ();
         let market = undefined;
```

Comment:
`fetchClosedOrders`consistently returns `System Error` so I think it might be related to our code

```
Node.js: v17.9.0
CCXT v2.2.35
bybit.fetchClosedOrders (BTC/USDT:USDT, , 1)
ExchangeError bybit {"retCode":10016,"retMsg":"System error. Please try again later.","result":{},"retExtInfo":{},"time":1669718855914}
---------------------------------------------------
[ExchangeError] bybit {"retCode":10016,"retMsg":"System error. Please try again later.","result":{},"retExtInfo":{},"time":1669718855914}

    at throwExactlyMatchedException  js/base/Exchange.js:2257              throw new exact[string] (message);                                              
    at handleErrors                  js/bybit.js:7140                      this.throwExactlyMatchedException (this.exceptions['exact'], errorCode, feedbac…
    at                               js/base/Exchange.js:598               const skipFurtherErrorHandling = this.handleErrors (response.status, response.s…
    at processTicksAndRejections     node:internal/process/task_queues:96                                                                                  
    at async timeout                 js/base/functions/time.js:181         return await Promise.race ([promise, expires.then (() => { throw new TimedOut (…
    at fetch2                        js/base/Exchange.js:1901              return await this.fetch (request['url'], request['method'], request['headers'],…
    at request                       js/base/Exchange.js:1905              return await this.fetch2 (path, api, method, params, headers, body, config, con…
    at fetchUnifiedMarginOrders      js/bybit.js:4321                      const response = await this.privateGetUnifiedV3PrivateOrderList (this.extend (r…
    at fetchOrders                   js/bybit.js:4473                      return await this.fetchUnifiedMarginOrders (symbol, since, limit, params);      
    at fetchClosedOrders             js/bybit.js:4559                      return await this.fetchOrders (symbol, since, limit, this.extend (request, para…

bybit {"retCode":10016,"retMsg":"System error. Please try again later.","result":{},"retExtInfo":{},"time":1669718855914}
```

Reply:
They only accept single status, shall we add another status in params?

Reply:
@carlosmiei closed orders mean the status is either filled or cancelled. I only use cancelled in unified margin, just wondering we can add status in params ('{"status":"Filled"}')

Reply:
@sc0Vu obviously, we have to return both (closed and filled) otherwise it's an incomplete `fetchClosedOrders` implementation, but will adding the other status in params work? I'm not sure I understood tbh